content/en/community/hackathons/usoc23/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Unikraft Summer of Code 2023"
+date: 2023-04-12T01:10:00+02:00
+draft: false
+collapsible: true
+description: Unikraft Summer of Code 2023, July 4 - 15, 2023
+---
+
+## Unikraft Summer of Code 2023
+
+Unikraft Summer of Code 2023 (*USoC'23*) is a **free and virtual** unikernel and library operating systems workshop held by members of the Unikraft community.
+
+The two-week event holds a number of starter tutorials, workshops and lessons on how to get build Unikraft unikernels for production systems.
+
+The workshop will be hands-on and will take place for six days, between July 4 and July 15, 2023, 4pm-7pm CEST, in English, virtually via [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
+Topics include building unikernels, benchmarking, debugging, porting applications, virtualization and platform specifics.
+The workshops will be followed by a full day hackathon which will be held live in `TBA` at [University POLITEHNICA of Bucharest](https://upb.ro), with the possibility to also join online via [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
```

Comment:
Update TBA

Reply:
Should it be room PR706?

Reply:
Mention it will be online (this is the default). For those how can make it, we will also organize the hackathon in room PR706.